focus-ios/Blockzilla/BrowserViewController.swift

```diff
@@ -20,12 +20,12 @@ class BrowserViewController: UIViewController {
         isTrackingEnabled: {
             Settings.getToggle(.trackingProtection)
         })
-    private lazy var webViewController: WebViewController = {
+    private lazy var webViewController: LegacyWebViewController = {
         var menuAction = WebMenuAction.live
         menuAction.openLink = { url in
             self.submit(url: url, source: .action)
         }
-        return WebViewController(trackingProtectionManager: trackingProtectionManager, webMenuAction: menuAction)
+        return LegacyWebViewController(trackingProtectionManager: trackingProtectionManager, webMenuAction: menuAction)
     }()
     private let webViewContainer = UIView()
```

Comment:
I think we can name this to legacy as well, as we'll have an EngineView instead as a container 🤔 
```suggestion
    private let legacyWebViewContainer = UIView()
```